extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -101,6 +101,13 @@ private function getUserById( $userId ) {
 			throw new \InvalidArgumentException( self::NO_USER_MATCH_ERROR );
 		}
 
+		$user->load();
```

Comment:
It seems counterintuitive, but the user class loads user info when a property is accessed, rather than when it is created. For example take the implementation of `getName()`, which this code was drawn from. https://github.com/Wikia/app/blob/dev/includes/User.php#L1817